class-tgm-plugin-activation.php

```diff
@@ -534,7 +534,9 @@ public function admin_init() {
 
 				global $tab, $body_id;
 				$body_id = 'plugin-information';
+				// @codingStandardsIgnoreStart
```

Comment:
This is an alternative way to get round that one issue where we were considering excluding the rule or lowering the severity.
Now the rule is still being used, just not for that one line.